src/app/page.tsx

```diff
@@ -1,12 +1,15 @@
-"use client"
-import "./style.css"
+"use client";
+import Image from "next/image";
+import imageLogoMarquei from "../app/LOGO_MARQUEI.jpeg";
+import "./style.css";
 
 export default function Home() {
   return (
-    <button onClick={function(){
-      alert("Olá mundo!")
-    }}>
-      Clique aqui!
-    </button>
+    <div>
+      <header>
+        <Image src={imageLogoMarquei} alt="The logo Marquei" />
+        <h2 className="show">Nome</h2>
```

Comment:
```suggestion
        <span className="show">Nome</span>
```

Acho que aqui a gente não deveria usar headers nos nomes. Eles não tem tanta importância assim pro conteúdo e nem representam seções ou itens. Então eu usaria apenas simples spans aqui.

Reply:
Alterações feitas

Reply:
![image](https://github.com/goislimat/marquei-web/assets/133810168/a326b7b8-ef47-4915-ada0-2988111c6d85)

Reply:
![Captura de tela 2024-01-31 163204](https://github.com/goislimat/marquei-web/assets/133810168/d2d53128-5bf4-4160-9bc8-2537a9886e01)

Reply:
![Captura de tela 2024-01-31 163204 1](https://github.com/goislimat/marquei-web/assets/133810168/3a83e025-4fc4-4677-82bb-ab463cbb0b63)